BitwardenShared/UI/Vault/VaultItem/CipherItemState.swift

```diff
@@ -107,6 +107,13 @@ struct CipherItemState: Equatable {
         self
     }
 
+    /// Whether or not this item can be deleted by the user.
+    var canBeDeleted: Bool {
+        guard !collectionIds.isEmpty else { return true }
+        let cipherCollections = collections.filter { collectionIds.contains($0.id ?? "") }
+        return cipherCollections.map(\.manage).contains(true)
```

Comment:
🎨 I think it's better to use a `contains` directly here so you don't need to traverse the whole `collections` sequence potentially three times (`filter` + `map` + `contains`) if `manage = true` is never valid or is the last one. Additionally, I feel it's simpler.
```suggestion
        return collections.contains { collection in
            collectionIds.contains(collection.id ?? "") && collection.manage
        }
```

Reply:
TIL `.contains` has a closure variation! Yeah, doing it that way is probably better.